src/pages/base/wallets/components/WalletInfo.js

```diff
@@ -58,23 +59,42 @@ function WalletInfo(props) {
     },
     {
       title: "health rate",
-      bigValue: health_rate,
+      bigValue: (
+        <Row className="mb3">
+          <Col className="d-flex align-items-center">
+            <div>{health_rate}</div>
+          </Col>
+          <Col className="d-flex align-items-center">
+            {data.user_mode === 1 ? (
+              <Badge style={{ fontSize: "0.8rem" }} color="primary">
+                e-mode
+              </Badge>
+            ) : null}
+          </Col>
+        </Row>
+      ),
     },
     {
       title: "risk",
-      normalValue: (
-        <>
-          {data.protection_score ? (
-            <Badge
-              id="riskBadge"
-              color={badgeColorMap[data.protection_score.protection_score]}
-            >
-              {data.protection_score.protection_score} risk
-            </Badge>
-          ) : (
-            "-"
-          )}
-        </>
+      bigValue: (
+        <Row className="mb3">
+          <Col className="d-flex align-items-center">
+            <div style={{ visibility: "hidden" }}>-</div>
+          </Col>
+          <Col className="d-flex align-items-center">
+            {data.protection_score ? (
+              <Badge
+                style={{ fontSize: "0.8rem" }}
+                id="riskBadge"
+                color={badgeColorMap[data.protection_score.protection_score]}
+              >
+                {data.protection_score.protection_score} risk
+              </Badge>
+            ) : (
+              "-"
+            )}
+          </Col>
+        </Row>
```

Comment:
Replace this part with
```suggestion
        <div className="lh-sm">
          {data.protection_score ? (
            <Badge
              style={{ fontSize: "0.8rem" }}
              id="riskBadge"
              color={badgeColorMap[data.protection_score.protection_score]}
              className="align-middle"
            >
              {data.protection_score.protection_score} risk
            </Badge>
          ) : (
            "-"
          )}
        </div>
```